FSharp.AutoComplete/Program.fs

```diff
@@ -617,6 +641,36 @@ module internal Main =
 
               main state
 
+          | SymbolUse ->
+              let symboluses =
+                  async {
+                      let! symboluse = tyRes.GetSymbol(line, col, lineStr)
+                      if symboluse.IsNone then return None else
+                      let! symboluses = tyRes.GetUsesOfSymbolInFile symboluse.Value.Symbol
+                      return Some {
+                        Name = symboluse.Value.Symbol.DisplayName
+                        Uses =
+                          [ for su in symboluses do
+                              yield { StartLine = su.RangeAlternate.StartLine
+                                      StartColumn = su.RangeAlternate.StartColumn + 1
+                                      EndLine = su.RangeAlternate.EndLine
+                                      EndColumn = su.RangeAlternate.EndColumn + 1
```

Comment:
Yes, I guess you posted at the same time as me ;-)

I would be happy if everything returned 1-based so at least FSAC is consistent even if FCS isn't.

@7sharp9 do you adjust the results you get from `GetNavigationItems` for the pathed document support as well to be 1-based?